src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.ts

```diff
@@ -127,7 +180,149 @@ export const transformOktaVerifyEnrollPoll: IdxStepTransformer = ({ transaction,
+          },
+        } as DescriptionElement,
+      );
+    } else {
+      sameDeviceOVElements.push(
+        {
+          type: 'Description',
+          noMargin: true,
+          options: {
+            content: loc(
+              'oie.enroll.okta_verify.setup.customUri.makeSureHaveOVToContinue', 'login',
+            ),
+          },
+        } as DescriptionElement,
+      );
+    }
+    if (deviceMap.platformLC) {
+      let platformSupported = true;
+      let svgIcon;
+      switch (deviceMap.platformLC) {
+        case 'android':
+          svgIcon = GoogleStoreIcon;
+          break;
+
+        case 'ios':
+          svgIcon = AppleStoreIcon;
+          break;
+
+        case 'windows':
+          svgIcon = WindowsStoreIcon;
+          break;
+
+        case 'osx':
+          svgIcon = OSXStoreIcon;
+          break;
+
+        default:
+          platformSupported = false;
+      }
+      if (platformSupported) {
+        sameDeviceOVElements.push(
+          {
+            type: 'ImageLink',
+            options: {
+              href: deviceMap.downloadHref,
+              altText: loc('enroll.oda.step3', 'login'),
+              alignment: 'center',
+              svgIcon,
+              dataSe: 'app-store-link',
+            },
+          } as ImageLinkElement,
+        );
+      }
+    }
+    sameDeviceOVElements.push(
+      {
+        type: 'Description',
+        noMargin: true,
+        options: {
+          content: loc(
+            'oie.enroll.okta_verify.setup.customUri.setup',
+            'login',
+            undefined,
+            { $1: { element: 'span', attributes: { class: 'strong' } } },
+          ),
+        },
+      } as DescriptionElement,
+    );
+
+    if (setupOVUrl) {
+      sameDeviceOVElements.push(
+        {
+          type: 'Button',
+          label: loc('oie.enroll.okta_verify.setup.title', 'login'),
+          options: {
+            step: '',
+            type: ButtonType.BUTTON,
+            variant: 'primary',
+            onClick: () => {
+              if (isAndroid()) {
+                Util.redirectWithFormGet(setupOVUrl);
+              } else {
+                window.location.assign(setupOVUrl);
+              }
+            },
+          },
+        } as ButtonElement,
+      );
+    }
+
+    if (showAnotherDeviceLink) {
+      sameDeviceOVElements.push(
+        {
+          type: 'Link',
+          options: {
+            // todo: refactor string parsing within OKTA-701606
+            label: loc(
+              deviceMap.isDesktop ? 'oie.enroll.okta_verify.setup.customUri.orOnMobile'
+                : 'oie.enroll.okta_verify.setup.customUri.orMobile',
+              'login',
+            ).replace('</$1>', '').replace('<$1>', '').trim(),
+            step: IDX_STEP.SELECT_ENROLLMENT_CHANNEL,
+            actionParams: {
+              'authenticator.channel': 'qrcode',
+              'authenticator.id': authenticator.id,
+            },
+          },
+        } as LinkElement,
```

Comment:
Can you give this a try? This is to prevent having the remove the `$1` from the string. 
```suggestion
        {
          type: 'TextWithActionLink',
          options: {
            content: loc(
              deviceMap.isDesktop ? 'oie.enroll.okta_verify.setup.customUri.orOnMobile'
                : 'oie.enroll.okta_verify.setup.customUri.orMobile',
              'login',
              undefined,
              {
                  $1: { element: 'a', attributes: { href: '#', class: 'orOnMobileLink' } },
              },
            ),
            contentClassname: 'orOnMobileLink',
            step: IDX_STEP.SELECT_ENROLLMENT_CHANNEL,
            actionParams: {
              'authenticator.channel': 'qrcode',
              'authenticator.id': authenticator.id,
            },
          },
        } as TextWithActionLinkElement,
```

Reply:
nice! thanks! please see https://github.com/okta/okta-signin-widget/pull/3526/commits/f98e3e2cea87b75c09bc2390574156208b5b5826